config/envs/default.py

```diff
@@ -51,6 +53,9 @@ class DefaultConfig:
 
     # Assesment store endpoints
     ASSESSMENT_STORE_API_HOST = CommonConfig.ASSESSMENT_STORE_API_HOST
+    APPLICATION_OVERVIEW_ENDPOINT = (
+        "/application_overviews/{fund_id}/{round_id}"
+    )
 
     ASSESSMENT_SCORE_JUST_ENDPOINT = (
```

Comment:
Can we spell this out fully i.e. `JUSTIFICATION` or `SCORES`?

Reply:
Done